internal/storage/entry.go

```diff
@@ -620,3 +620,13 @@ func removeDuplicates(l []string) []string {
 	slices.Sort(l)
 	return slices.Compact(l)
 }
+
+func removeEmpty(l []string) []string {
```

Comment:
Empty tags shouldn't be added to `entry.Tags` in the first place, instead of being added and then filtered, I think.

Reply:
Ok, but this is the same for the removeDuplicates part. I think these functions would not have to be executed in a storage module but in the feed parser directly but this would need a little refactoring and I don't want to do this in this PR but maybe in a dedicated PR.

Reply:
Please add a `TODO: this should happen in the feed parser instead of here` or something equivalent :)

Reply:
TODO comments are a bit ugly. I opened an issue ;) #2563